api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -3014,4 +3014,36 @@ public void getWorkspaceResources() {
     assertThat(dataSets).hasSize(1);
     compareDatasetMetadata(dataSets.get(0), dataSet);
   }
+
+  @Test
+  public void testMarkWorkspacePublished_ByUserWithWriterAccess() {
+    assertThrows(
+        ForbiddenException.class,
+        () -> {
+          Workspace ws = createWorkspace();
+          ws = workspacesController.createWorkspace(ws).getBody();
+          ws.setName("updated-name");
+          UpdateWorkspaceRequest request = new UpdateWorkspaceRequest();
+          request.setWorkspace(ws);
+          stubGetWorkspace(
+              ws.getNamespace(), ws.getId(), ws.getCreator(), WorkspaceAccessLevel.WRITER);
+          workspacesController.markWorkspaceAsFeatured(ws.getNamespace());
```

Comment:
it can be a little clearer to include only the offending line inside the  `assertThrows()`.  For example:
```
Workspace ws;
...
other setup
...
 assertThrows(
        ForbiddenException.class,
        () -> workspacesController.markWorkspaceAsFeatured())
```